code/game/area/ss13_areas.dm

```diff
@@ -1108,7 +1108,7 @@ NOTE: there are two lists of areas in the end of this file: centcom and station
 	sound_environment = SOUND_AREA_SMALL_ENCLOSED
 
 /area/engine/equipmentstorage
-	name = "\improper Engineering Equipment Storage"
+	name = "Engineering Equipment Storage"
```

Comment:
I am worried with the amount of extra side effects this will cause.

Reply:
It'll almost definitely have side effects, but all of them *should* be good ones since it's more gramatically correct now.